package/MDAnalysis/coordinates/TRC.py

```diff
@@ -0,0 +1,409 @@
+a PDB::
+
+    import MDAnalysis as mda
+    u = mda.Universe("topology.pdb", ["md_1.trc.gz","md_2.trc.gz"],
+    continuous=True)
+
+.. Note::
+   The GROMOS trajectory format organizes its data in blocks. A block starts
+   with a blockname in capital letters (example: POSITIONRED) and ends with
+   a line containing only ''END''. Only the TITLE-block at the beginning of
+   each file is mandatory, others blocks can be chosen depending on the task.
+
+   The trajectory format as well as all permitted blocks and their data are
+   documented in the GROMOS Manual Vol. 4, chapter 2 and 4.
+   The manual can be downloaded here:
+   https://gromos.net/gromos11_pdf_manuals/vol4.pdf
+
+This reader is designed to read the blocks "TIMESTEP", "POSITIONRED" and
+"GENBOX" from the trajectory which covers the standard trajectories of most
+simulations . If a trajectory includes other blocks, a warning is served
+and the blocks are ignored.
+
+.. Note::
+   MDAnalysis requires the blocks to be in the same order for each frame
+   and ignores non-supported blocks.
+
+
+
+Classes
+-------
+
+.. autoclass:: TRCReader
+   :members:
+
+"""
+
+import pathlib
+import errno
+import warnings
+import numpy as np
+
+from . import base
+from .timestep import Timestep
+from ..lib import util
+from ..lib.util import cached, store_init_arguments
+
+import logging
+
+logger = logging.getLogger("MDAnalysis.coordinates.GROMOS11")
+
+
+class TRCReader(base.ReaderBase):
+    """Coordinate reader for the GROMOS11 format"""
+
+    format = "TRC"
+    units = {"time": "ps", "length": "nm"}
+    _Timestep = Timestep
+
+    SUPPORTED_BLOCKS = ["TITLE", "TIMESTEP", "POSITIONRED", "GENBOX"]
+    NOT_SUPPORTED_BLOCKS = [
+        "POSITION",
+        "REFPOSITION",
+        "VELOCITY",
+        "VELOCITYRED",
+        "FREEFORCE",
+        "FREEFORCERED",
+        "CONSFORCE",
+        "CONSFORCERED",
+        "SHAKEFAILPOSITION",
+        "SHAKEFAILPREVPOSITION",
+        "LATTICESHIFTS",
+        "COSDISPLACEMENTS",
+        "STOCHINT",
+        "NHCVARIABLES",
+        "ROTTRANSREFPOS",
+        "PERTDATA",
+        "DISRESEXPAVE",
+        "JVALUERESEXPAVE",
+        "JVALUERESEPS",
+        "JVALUEPERSCALE",
+        "ORDERPARAMRESEXPAVE",
+        "XRAYRESEXPAVE",
+        "LEUSBIAS",
+        "LEUSBIASBAS",
+        "ENERGY03",
+        "VOLUMEPRESSURE03",
+        "FREEENERGYDERIVS03",
+        "BFACTOR",
+        "AEDSS",
+    ]
+    
+    @store_init_arguments
+    def __init__(self, filename, convert_units=True, **kwargs):
+        super(TRCReader, self).__init__(filename, **kwargs)
+
+        # GROMOS11 trajectories are usually either *.trc or *.trc.gz.
+        # (trj suffix can come up when trajectory obtained from clustering)
+        ext = pathlib.Path(self.filename).suffix
+        if (ext[1:] == "trc") or (ext[1:] == "trj"):
+            self.compression = None
+        else:
+            self.compression = ext[1:]
```

Comment:
I can't see where else this attribute is used

Reply:
It's not used but should have been the documented optional attribute `compressed` of the trajectory reader class that might be accessed by the user.
I corrected it from `compression` to `compressed`

The readers XYZ and H5MD also use the attribute `compression`. Might be worth to check out, if that is also a typo.